includes/plugins/class-woocommerce-memberships.php

```diff
@@ -134,6 +134,19 @@ function ( $list_id ) use ( $list_ids ) {
 	public static function handle_membership_status_change( $user_membership, $old_status, $new_status ) {
 		Newspack_Newsletters_Logger::log( 'Membership status changed to ' . $new_status );
 
+		$user = $user_membership->get_user();
+		if ( ! $user ) {
+			return;
+		}
+		$user_email = $user->user_email;
+
+		$log_data = [
+			'old_status'    => $old_status,
+			'new_status'    => $new_status,
+			'membership_id' => $user_membership->get_id(),
+		];
+		Newspack_Newsletters_Logger::remote_log( 'newsletters_membership_hadnling', 'Membership status changed', $user_email, [ 'data' => $log_data ] );
```

Comment:
Typo: here and in three other instances.

```suggestion
		Newspack_Newsletters_Logger::remote_log( 'newsletters_membership_handling', 'Membership status changed', $user_email, [ 'data' => $log_data ] );
```